Xamarin.Essentials/Accelerometer/Accelerometer.uwp.cs

```diff
@@ -20,7 +18,8 @@ public static partial class Accelerometer
 
         internal static void PlatformStart(SensorSpeed sensorSpeed)
         {
-            sensor = DefaultSensor;
+            var sensor = DefaultSensor;
```

Comment:
Can't do this, it will cause an error and actually not stop the sensor. Have to keep around a reference :(

Reply:
Let's add a comment on it so we don't accidentally move it.